src/components/Tool/Tool.js

```diff
@@ -35,37 +63,34 @@ class Tool extends PureComponent {
     }
 
     handleChangeMonth(month) {
-        this.setState({month});
+        // TODO: Refactor so that months are everywhere 0-based
```

Comment:
Eeek yeah. Where are they 1-based and where are they 0-based?

Reply:
0-based because most date objects in JS (including the builtin Date, and `moment`) use a 0-based date, which is intensely annoying because humans don't. But this I think is easily fixed with a mod to my MonthSelector component. Then everyone will play together happily.